src/ui/public/utils/find_by_param.js

```diff
@@ -0,0 +1,13 @@
+define(function (require) {
```

Comment:
Pulled this out of `src/ui/public/filter_bar/filter_bar_click_handler.js` into a `utils/` function so it could be reused.